docs/ortho-config-v0-7-0-migration-guide.md

```
@@ -0,0 +1,226 @@
+```toml
+ortho_config = { version = "0.7.0", features = ["yaml"] }
+ortho_config_macros = "0.7.0"
+```
+
+1. Update every `ortho_config` and `ortho_config_macros` dependency to `0.7.0`.
+1. Keep format features (`toml`, `json5`, `yaml`) on `ortho_config` as before.
+1. If you disable default features, enable `serde_json` explicitly when you
+   want `cli_default_as_absent` or selected-subcommand merging helpers. The
+   default feature set already enables it.
+1. Expect new transitive dependencies (`fluent-bundle`, `fluent-syntax`,
+   `unic-langid`, and `tracing`) to land with v0.7.0, as they power CLI
```

markdownlint-cli2 (0.21.0)

[warning] Ordered list item prefix
Expected: 2; Actual: 1; Style: 1/2/3

(MD029, ol-prefix)